src/app/static/src/tests/components/plots/timeSeries/utils.test.ts

```diff
@@ -0,0 +1,834 @@
+import {
```

Comment:
I think this might be the only test which is significantly different from what was here before. The structure of the data fed to plotly has changed a little. Particularly in the last test in this file. Compare with tests https://github.com/mrc-ide/hint/pull/933/files#diff-8cb0738a8cb58914fdeea490531b292ecc16df44d269c978a51fe5e8c114d181L1